lib/zssAuth.js

```diff
@@ -69,6 +72,12 @@ ZssAuthenticator.prototype = {
         body: request.body
       };
       request.zluxData.webApp.callRootService("login", options).then((response) => {
+        if (response.statusCode == HTTP_STATUS_PRECONDITION_REQUIRED) {
+          sessionState.authenticated = false;
+          delete sessionState.zssUsername;
+          delete sessionState.zssCookies;
+          resolve({ success: false, reason: 'Expired Password'});
```

Comment:
add new property to resolves: "canChangePw": true/false
That way you can mark which plugins are capable of this for future use in the browser.